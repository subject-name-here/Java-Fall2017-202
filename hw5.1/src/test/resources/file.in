-5
6
8
-9
12 33
3.4
null
6
