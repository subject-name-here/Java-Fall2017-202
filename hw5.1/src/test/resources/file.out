25
36
64
81
null
null
null
36
